index.html

```diff
@@ -45,5 +45,120 @@ <h2>LET'S CONNECT</h2>
+      </div> 
+
+</section>
+<section class="cardworks-three">
+  <img src="./images/Snapshoot Portfolio (3).png" alt="" class="snapshoot">
+  <div class="left-blocktwo"></div>
+  <h5 class="headings">Tonic</h5>
+<img src="./images/Frame 2.png" alt="" class="frame">
+  <p>
+   A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+ <img src="./images/Tags.png" alt="" class="tag">
+      <div class="buttons">
+      <button class="press">See project</button>  
+      </div>
+</section>
+<section class="cardworks-four">
+  <img src="./images/Snapshoot Portfolio (4).png" alt="" class="snapshoot">
+  <div class="left-blockthree"></div>
+  <h6 class="headings">Multi-Post Stories</h6> 
+  <img src="./images/Frame 2.png" alt="" class="frame">
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. 
+  </p>
+  <img src="./images/Tags.png" alt="" class="tag">
+      <div class="buttons">
+       <button class="press">See project</button> 
+      </div>
+</section>
+<section class="about-me">
+        <div>
+            <div class="about">About
+                Myself</div>
+                <p class="about-paragraph">Hello I’m a software developer! I can help you build a product , feature or  website Look through some of my work and experience! If you like what you see and  have a project you need coded,don’t hestiate to contact me.</p>
+                <h2>LET'S CONNECT</h2>
+                <ul class="tags">
+                    <li>
+                        <a href="#"><img src="./images/Vector (6).png" alt="github"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="./images/Vector (2).png" alt="linkedin"></a>
+                        <a href="#"><img src="./images/Vector (2).png" alt="linkedin"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img class="thumbs-up" src="./images/Vector (5).png" alt="thumbs-up"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="./images/Vector (1).png" alt="twitter"></a>
+                    </li> <li>
+                        <a href="#"><img src="./images/Vector (4).png" alt="Vector"></a>
+                    </li>
+                </ul>
+                <button type="submit" class="start">Get my resume</button>
+        </div>
+        <ul class="language">
+            <li>
+                <p class="languages">Languages</p>
+            </li>
+            
+        </ul>
+   <ul>
+        <li class="lists">
+          <a href="#"><img src="./images/Frame 9.png" alt="javascript"></a>
+        </li>
+         <li class="lists">
+          <a href="#"><img src="./images/Frame 8.png" alt="html"></a>
+         </li>
+         <li class="lists">
+          <a href="#"><img src="./images/Frame 11.png" alt="css"></a>
+         </li>
+         </ul>
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/brendalutiku/Portfolio/pull/2#discussion_r1154248069) please make sure to style the skills list according to the Figma design guidelines 😄 Also, please make sure to include the `>` icons on the right edge 🛠 See the images below for more clarity 👇

You design before 🔽 | Your design now 🔽 | Figma design 🔽
---| ---| ---|
![image](https://user-images.githubusercontent.com/106119105/229164384-b7283c9e-d331-4c18-b52e-1a5ade678004.png) | ![image](https://user-images.githubusercontent.com/106119105/229164312-39dd13e5-0457-4fd0-9888-82cd28797f9a.png) | ![image](https://user-images.githubusercontent.com/106119105/229164464-acd3705b-875c-4e3a-8996-39e5fc3edd81.png)